--- anki-2.1.49/rules_rust/bindgen/raze/remote/BUILD.atty-0.2.14.bazel
+++ anki-2.1.49_arm/rules_rust/bindgen/raze/remote/BUILD.atty-0.2.14.bazel
@@ -61,6 +61,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/bindgen/raze/remote/BUILD.clap-2.33.3.bazel
+++ anki-2.1.49_arm/rules_rust/bindgen/raze/remote/BUILD.clap-2.33.3.bazel
@@ -72,6 +72,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/bindgen/raze/remote/BUILD.libloading-0.6.3.bazel
+++ anki-2.1.49_arm/rules_rust/bindgen/raze/remote/BUILD.libloading-0.6.3.bazel
@@ -63,6 +63,7 @@ cargo_build_script(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -118,6 +119,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/bindgen/raze/remote/BUILD.libloading-0.7.0.bazel
+++ anki-2.1.49_arm/rules_rust/bindgen/raze/remote/BUILD.libloading-0.7.0.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/cargo/cargo_build_script_runner/bin.rs
+++ anki-2.1.49_arm/rules_rust/cargo/cargo_build_script_runner/bin.rs
@@ -65,6 +65,7 @@ fn run_buildrs() -> Result<(), String> {
         .env("OUT_DIR", out_dir_abs)
         .env("CARGO_MANIFEST_DIR", manifest_dir)
         .env("RUSTC", rustc)
+        .env("PROTOC", "/usr/bin/protoc")
         .env("RUST_BACKTRACE", "full");

     for dep_env_path in input_dep_env_paths.iter() {
--- anki-2.1.49/rules_rust/examples/complex_sys/raze/remote/BUILD.git2-0.13.12.bazel
+++ anki-2.1.49_arm/rules_rust/examples/complex_sys/raze/remote/BUILD.git2-0.13.12.bazel
@@ -97,6 +97,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
             "@rules_rust//rust/platform:i686-unknown-linux-gnu",
--- anki-2.1.49/rules_rust/examples/complex_sys/raze/remote/BUILD.jobserver-0.1.21.bazel
+++ anki-2.1.49_arm/rules_rust/examples/complex_sys/raze/remote/BUILD.jobserver-0.1.21.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/examples/complex_sys/raze/remote/BUILD.libgit2-sys-0.12.18+1.1.0.bazel
+++ anki-2.1.49_arm/rules_rust/examples/complex_sys/raze/remote/BUILD.libgit2-sys-0.12.18+1.1.0.bazel
@@ -74,6 +74,7 @@ cargo_build_script(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -130,6 +131,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/examples/complex_sys/raze/remote/BUILD.libssh2-sys-0.2.20.bazel
+++ anki-2.1.49_arm/rules_rust/examples/complex_sys/raze/remote/BUILD.libssh2-sys-0.2.20.bazel
@@ -79,6 +79,7 @@ cargo_build_script(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -137,6 +138,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.httpbis-0.7.0.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.httpbis-0.7.0.bazel
@@ -79,6 +79,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.iovec-0.1.4.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.iovec-0.1.4.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.mio-0.6.21.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.mio-0.6.21.bazel
@@ -68,6 +68,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.mio-uds-0.6.7.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.mio-uds-0.6.7.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.net2-0.2.33.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.net2-0.2.33.bazel
@@ -62,6 +62,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.parking_lot_core-0.6.2.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.parking_lot_core-0.6.2.bazel
@@ -63,6 +63,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.tokio-0.1.22.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.tokio-0.1.22.bazel
@@ -130,6 +130,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/proto/raze/remote/BUILD.tokio-tls-api-0.1.22.bazel
+++ anki-2.1.49_arm/rules_rust/proto/raze/remote/BUILD.tokio-tls-api-0.1.22.bazel
@@ -60,6 +60,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
             "@rules_rust//rust/platform:i686-unknown-linux-gnu",
--- anki-2.1.49/rules_rust/rust/platform/platform.bzl
+++ anki-2.1.49_arm/rules_rust/rust/platform/platform.bzl
@@ -31,6 +31,7 @@ _SUPPORTED_T2_PLATFORM_TRIPLES = [
     "aarch64-linux-android",
     "aarch64-unknown-linux-gnu",
     "arm-unknown-linux-gnueabi",
+    "arm-unknown-linux-gnueabihf",
     "i686-linux-android",
     "i686-unknown-freebsd",
     "powerpc-unknown-linux-gnu",
--- anki-2.1.49/rules_rust/rust/repositories.bzl
+++ anki-2.1.49_arm/rules_rust/rust/repositories.bzl
@@ -17,6 +17,7 @@ DEFAULT_STATIC_RUST_URL_TEMPLATES = ["ht
 DEFAULT_TOOLCHAIN_TRIPLES = {
     "aarch64-apple-darwin": "rust_darwin_aarch64",
     "aarch64-unknown-linux-gnu": "rust_linux_aarch64",
+    "arm-unknown-linux-gnueabihf": "rust_linux_arm",
     "x86_64-apple-darwin": "rust_darwin_x86_64",
     "x86_64-pc-windows-msvc": "rust_windows_x86_64",
     "x86_64-unknown-freebsd": "rust_freebsd_x86_64",
--- anki-2.1.49/rules_rust/tools/rust_analyzer/raze/remote/BUILD.atty-0.2.14.bazel
+++ anki-2.1.49_arm/rules_rust/tools/rust_analyzer/raze/remote/BUILD.atty-0.2.14.bazel
@@ -61,6 +61,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/tools/rust_analyzer/raze/remote/BUILD.clap-2.33.3.bazel
+++ anki-2.1.49_arm/rules_rust/tools/rust_analyzer/raze/remote/BUILD.clap-2.33.3.bazel
@@ -72,6 +72,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/tools/rustfmt/BUILD.bazel
+++ anki-2.1.49_arm/rules_rust/tools/rustfmt/BUILD.bazel
@@ -9,6 +9,7 @@ alias(
     actual = select({
         "@rules_rust//rust/platform:aarch64-apple-darwin": "@rust_darwin_aarch64//:rustfmt_bin",
         "@rules_rust//rust/platform:aarch64-unknown-linux-gnu": "@rust_linux_aarch64//:rustfmt_bin",
+        "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf": "@rust_linux_arm//:rustfmt_bin",
         "@rules_rust//rust/platform:x86_64-apple-darwin": "@rust_darwin_x86_64//:rustfmt_bin",
         "@rules_rust//rust/platform:x86_64-pc-windows-msvc": "@rust_windows_x86_64//:rustfmt_bin",
         "@rules_rust//rust/platform:x86_64-unknown-linux-gnu": "@rust_linux_x86_64//:rustfmt_bin",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.atty-0.2.14.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.atty-0.2.14.bazel
@@ -61,6 +61,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.buf_redux-0.8.4.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.buf_redux-0.8.4.bazel
@@ -61,6 +61,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-pc-windows-msvc",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.curl-0.4.34.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.curl-0.4.34.bazel
@@ -69,6 +69,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
             "@rules_rust//rust/platform:i686-unknown-linux-gnu",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.curl-sys-0.4.39+curl-7.74.0.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.curl-sys-0.4.39+curl-7.74.0.bazel
@@ -64,6 +64,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
             "@rules_rust//rust/platform:i686-unknown-linux-gnu",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.dirs-1.0.5.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.dirs-1.0.5.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.filetime-0.2.13.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.filetime-0.2.13.bazel
@@ -60,6 +60,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.getrandom-0.1.15.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.getrandom-0.1.15.bazel
@@ -73,6 +73,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.4.6.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.4.6.bazel
@@ -68,6 +68,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.5.6.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.5.6.bazel
@@ -77,6 +77,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.6.5.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.6.5.bazel
@@ -85,6 +85,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.7.3.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rand-0.7.3.bazel
@@ -81,6 +81,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rand_os-0.1.3.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rand_os-0.1.3.bazel
@@ -60,6 +60,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.rayon-core-1.9.0.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.rayon-core-1.9.0.bazel
@@ -66,6 +66,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.socket2-0.3.19.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.socket2-0.3.19.bazel
@@ -59,6 +59,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.tempfile-3.1.0.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.tempfile-3.1.0.bazel
@@ -62,6 +62,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
--- anki-2.1.49/rules_rust/wasm_bindgen/raze/remote/BUILD.wait-timeout-0.2.0.bazel
+++ anki-2.1.49_arm/rules_rust/wasm_bindgen/raze/remote/BUILD.wait-timeout-0.2.0.bazel
@@ -61,6 +61,7 @@ rust_binary(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -110,6 +111,7 @@ rust_binary(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -159,6 +161,7 @@ rust_binary(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
@@ -206,6 +209,7 @@ rust_library(
             "@rules_rust//rust/platform:aarch64-linux-android",
             "@rules_rust//rust/platform:aarch64-unknown-linux-gnu",
             "@rules_rust//rust/platform:arm-unknown-linux-gnueabi",
+            "@rules_rust//rust/platform:arm-unknown-linux-gnueabihf",
             "@rules_rust//rust/platform:i686-apple-darwin",
             "@rules_rust//rust/platform:i686-linux-android",
             "@rules_rust//rust/platform:i686-unknown-freebsd",
